doc/sphinxext/ipython_sphinxext/ipython_directive.py

```diff
@@ -127,7 +127,11 @@
 from sphinx.util.compat import Directive
 
 # Our own
-from IPython import Config, InteractiveShell
+try:
+    from traitlets.config import Config
+except ImportError:
```

Comment:
They put a warning on the module (https://github.com/ipython/ipython/blob/master/IPython/config.py), but not on the object in the top-level namespace apparantly.

@takluyver is this an oversight of IPython or on purpose?